### PR TITLE
Added object operations

### PR DESCRIPTION
**Pull request by ketanarlulkar**
# Problem Statement
-  Added object operations with workload file

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [ ] JIRA number/GitHub Issue added to PR
-  [ ] PR is self reviewed
-  [ ] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

File: config/s3/s3api/random_size_object.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+test_1:
+  TEST_ID: TEST-35678
+  object_size:
+    start : 0Kib
+    end : 100Kib
+  result_duration: 1h
+  sessions_per_node: 1
+  tool: s3api
+  operation: object
```
Comment (DhobaleVishal) [inline config/s3/s3api/random_size_object.yaml]: Operation should have explanatory.

File: corio.py [hunk 2]
```diff
@@ -39,29 +39,22 @@
 import schedule
 
 from config import S3_CFG
-from scripts.s3.s3api import bucket_operations
+from scripts.s3.s3api import bucket_operations, object_operations, object_range_read_operations
```
Comment (DhobaleVishal) [inline corio.py:42]: Imports should be in separate line.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

File: corio.py (hunk 2 above)
Comment (ketanarlulkar) [reply corio.py:42]: This is allowed in [pep8](https://peps.python.org/pep-0008/#imports)

**Review by kulrk — APPROVED** [review]

File: config/s3/s3api/random_size_object.yaml (hunk 1 above)
Comment (ketanarlulkar) [reply config/s3/s3api/random_size_object.yaml]: Looks self-explanatory and simple. Please suggest.

**Review by VikasKumarC — APPROVED** [review]

File: config/s3/s3api/random_size_object.yaml (hunk 1 above)
Comment (DhobaleVishal) [reply config/s3/s3api/random_size_object.yaml:9]: Team is okay with this so resolving this comment.

**Review by DhobaleVishal — APPROVED** [review]

**Pull request merged**